trimmerdecorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'Decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    @name.correct_name[0, 10]
+  end
```

Comment:
- Please make sure also that The name has a maximum of 10 characters. If it's longer it should trim the word. :+1: 

```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```